src/renderer_vk.cpp

```diff
@@ -1080,7 +1080,11 @@ VK_IMPORT_DEVICE
 			);
 	}
 
+#ifndef VK_MAX_DESCRIPTOR_SETS
```

Comment:
Holly ran into this last week too. I'm thinking of a better solution like growing the descriptor pool more dynamically, but in the short term this is fine